com.unity.ml-agents/Documentation~/index.md

```diff
@@ -1 +1,71 @@
-Please see the [ML-Agents README)(https://github.com/Unity-Technologies/ml-agents/blob/master/README.md)
\ No newline at end of file
+# About Unity MLAgents package
```

Comment:
is there a hyphen missing in "MLAgents"?

Reply:
Fixed.